### DISCUSSION TITLE
docs/v1.10.x/advance/fastpath/

### DISCUSSION THREAD

**Discussion by giscus** [kubeovn/kube-ovn#2013] [Announcements]
# docs/v1.10.x/advance/fastpath/

https://kubeovn.github.io/docs/v1.10.x/advance/fastpath/

**Comment by changluyi** [comment]
需要把kube_ovn_fastpath.ko , cp到/tmp/目录下，不然fastpath内核模块，会被kube-ovn-daemon周期给清理掉

**Reply by oilbeater** [reply]
已更新，多谢
